src/sets.jl

```diff
@@ -438,6 +438,18 @@ Base.isapprox(a::SOS2{T}, b::SOS2{T}; kwargs...) where T = isapprox(a.weights, b
 
 dimension(s::Union{SOS1, SOS2}) = length(s.weights)
 
+"""
+Set of (x,y) that satisfy the indicator constraint:
+    y ∈ 𝔹, y = 1 ==> a^T x ∈ S
+
+S can be one of `SmallerThan, GreaterThan, EqualTo`
```

Comment:
s/SmallerThan/LessThan

Documentation nits: 
 - use proper punctuation (e.g., end sentences with a period)
 - begin docstrings with the name of the type.
 - use backticks for things like `S`